### PR TITLE
add featEng

### PR DESCRIPTION
**Pull request by Kafkaese**
I added this awesome feature Engienerring script

### CODE REVIEW

**Comment by Kafkaese** [comment]
@davydw  can u have a look

**Comment by davydw** [comment]
@Kafkaese yeah sure, gimme 2 sec

File: featEng.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('hello')
```
Comment (davydw) [inline featEng.py:1]: this is perfect, amazing hello print

**Review by davydw — APPROVED** [review]
LGTM

**Pull request merged**